SmartDeviceLink/public/SDLMacros.h

```diff
@@ -9,15 +9,6 @@
 #ifndef SDLMacros_h
 #define SDLMacros_h
 
-// Resolves issue of pre-xcode 8 versions due to NS_STRING_ENUM unavailability.
-#ifndef SDL_SWIFT_ENUM
-    #if __has_attribute(swift_wrapper)
-        #define SDL_SWIFT_ENUM NS_STRING_ENUM
-    #else
-        #define SDL_SWIFT_ENUM
-    #endif
-#endif
-
 #ifndef SDL_SUPPORTS_CLASS_PROPERTIES
```

Comment:
It looks like this is an iOS 8 shim as well?

Reply:
Fixed. Removed `SDL_SUPPORTS_CLASS_PROPERTIES` and deleted the SDLMacros.h file